nexong/admin.py

```diff
@@ -9,3 +9,10 @@
 admin.site.register(Meeting)
 admin.site.register(Family)
 admin.site.register(Student)
+admin.site.register(Partner)
+admin.site.register(Volunteer)
+admin.site.register(Comment)
```

Comment:
Please remove this line and the LessonEvaluation lines since they are not part of the DB anymore.